modules/schema/src/main/java/org/apache/ignite/internal/schema/SchemaManager.java

```diff
@@ -64,13 +65,8 @@ public class SchemaManager extends Producer<SchemaEvent, SchemaEventParameters>
     /** Initial version for schemas. */
     public static final int INITIAL_SCHEMA_VERSION = 1;
 
-    /**
-     * If this property is set to {@code true} then an attempt to get the configuration property directly from the meta storage will be
-     * skipped, and the local property will be returned.
-     * TODO: IGNITE-16774 This property and overall approach, access configuration directly through the Metostorage,
-     * TODO: will be removed after fix of the issue.
-     */
-    private final boolean getMetadataLocallyOnly = IgniteSystemProperties.getBoolean("IGNITE_GET_METADATA_LOCALLY_ONLY");
+    /** Schema history key predicate part. */
+    public static final String SCHEMA_STORE_PREDICATE = ".sch-hist:";
```

Comment:
```suggestion
    public static final String SCHEMA_STORE_PRRFIX = ".schema-history:";
```
Maybe `.schema-history.` prefix will be more convenient for parsing schema version number,
it will always be a number after the last dot.